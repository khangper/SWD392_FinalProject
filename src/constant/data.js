export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
