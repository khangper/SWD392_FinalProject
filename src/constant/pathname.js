export const PATH_NAME = {
  HOME: "/home",
  EXPLORE: "/explore",
  LIVE_OUTPUT: "/live_output/:id",
  ADD_LIVE_STREAM: "/add_livestream",
  ALL_LIVE_STREAM: "/all_livestream",
  ALL_INSTRUCTOR_LIST: "/allinstructorlist/:id",
  HELP: "/help",
  REPORT_HISTORY: "/reporthistory",
  SEND_FEEDBACK: "/sendfeedback",
  OTHER_INSTRUCTOR_VIEW: "/other-instructor-view/:id",
  MY_INSTRUCTOR_PROFILE: "/my-instructor-profile",
  SAVE_COURSES: "/save-courses",
  SETTING: "/setting",
  MY_COURSE: "/courses",
  LOGIN: "/login",
  FORGOT_PASSWORD: "/forgot-password",
  SIGN_UP: "/sign-up",
  SIGN_UP_STEP: "/SignUpStep",
  THANKS: "/thanks",
  ERROR_404: "/error404",
  INVOICE: "/invoice",
  PRESS: "/press",
  SEARCH_RESULT: "/search_result",
  PAID_MEMBERSHIP: "/paidmembership",
  ABOUT_US: "/about-us",
  OUR_BLOG: "/our-blog",
  BLOG_SINGLE_VIEW: "/blog-single-view",
  CHECKOUT: "/checkout",
  SHOPPING_CART: "/shoppingcart",
  HELP_VIEW_1: "/helpview1",
  HELP_VIEW_2: "/helpview2",
  COMPANY_DETAILS: "/company-details",
  CAREER: "/career",
  APPLY_JOB: "/apply-job",
  CONTACT_US: "/contactus",
  TERMS_OF_USE: "/termsofuse",
  CERTIFICATION_FILL_FORM: "/certificationfillform",
  CERTIFICATION_TEST: "/certificates-test",
  TEST_RESULT: "/test-result",
  MESSAGE: "/messages",
  COURSES_DETAIL_VIEW: "/courses-detail-view/:id",
  PREVIEWS: "/reviews",
  COMING_SOON: "/coming-soon",
  INS_DASHBOARD: "/ins-dashboard",
  PURCHASE_COURSE: "/PurchaseCourse",
  NOTIFICATON: "/notifications",
  DOWLOADCOURSE: "/dowloadcourse",
  CERTIFICATION_CENTER: "/certificationcenter",
  SITEMAP: "/sitemap",
  EARNING: "/earning",
  PAYOUT: "/payout",
  VERTIFICATION: "/vertification",
  MYCERTIFICATIONS: "/mycertifications",
  CREDIT: "/credits",
  STATEMENTIS: "/statementis",
  STATEMENTST: "/statementst",
  ANALYTICS: "/analytics",
  CREATE_NEW_COURSE: "/create-course",
  STD_DASHBOARD: "/student-dashboard",
  NOTIFICATIONSTUDENT: "/notificationsstudent",
};
