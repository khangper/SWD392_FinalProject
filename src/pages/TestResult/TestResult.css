.ts-tick-container {
    width: 100px; /* Kích thước của biểu tượng */
    height: 100px;
    background-color: rgb(12, 195, 12); /* Màu nền */
    display: flex;
    align-items: center;
    justify-content: center;
    border-radius: 50%; /* Bo góc nếu cần */
    margin-bottom: 15px;
  }

.ts-tick_icon {
    width: 20px;
    height: 35px;
    border: solid white;
    border-width: 0 5px 5px 0;
    transform: rotate(45deg);
  }

.ct-cross-container {
    padding-right: 3px;
    padding-bottom: 5px;
    width: 100px; /* Kích thước của biểu tượng */
    height: 100px;
    background-color: red; /* Màu nền */
    display: flex;
    align-items: center;
    justify-content: center;
    border-radius: 50%; /* Bo góc nếu cần */
    margin-bottom: 15px;
}
  
  .ct-cross_icon {
    width: 24px;
    height: 24px;
    position: relative;
  }
  
  .ct-cross_icon::before,
  .ct-cross_icon::after {
    
    content: '';
    position: absolute;
    width: 5px; /* Độ rộng của dấu X */
    height: 35px;
    background-color: white; /* Màu của dấu X */
  }
  
  .ct-cross_icon::after {
    transform: rotate(-45deg);
    top: 0;
    left: 50%;
  }
  
  .ct-cross_icon::before {
    transform: rotate(45deg);
    top: 0;
    left: 50%;
  }

.test_result-bg {
    display: flex;
    flex-direction: column;
}

  .test_result-left {
    display: flex;
    justify-content: center;
    align-items: center;
  }

.ts-number-container {
    width: 100px; /* Kích thước của biểu tượng */
    height: 100px;
    background-color:#57a4ff; /* Màu nền */
    display: flex;
    align-items: center;
    justify-content: center;
    border-radius: 50%; /* Bo góc nếu cần */
    margin-bottom: 15px;
}

.ts-number-container h4 {
    color: white;
    font-size: 24px;
    font-family: sans-serif;
}

.test_result-left {
    margin-top: 57px;
    display: flex;
  justify-content: center;
  align-items: center;
  gap: 75px; /* Khoảng cách giữa các biểu tượng */
}

.ts-box-icon {
    display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center;
}

.result_dt {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.result_dt p {
    font-size: 18px;
    font-weight: 600;
    font-family: 'Roboto', sans-serif;
    color: #333;
    margin-top: 0;
    text-align: center;
    line-height: 26px;
}

.result_dt p span {
    color: #686f7a;
    margin-left: 5px;
    font-weight: 400;
}

.test_result-content {
    margin-top: 57px;
    float: left;
    width: 100%;
    text-align: center;
    margin-bottom: 57px;
}

.test_result-content h2 {
    font-size: 42px;
    font-weight: 600;
    color: #333;
    font-family: 'Roboto', sans-serif;
    text-align: center;
    padding-bottom: 15px;
}

.test_result-content p {
    font-size: 20px;
    font-weight: 400;
    color: #686f7a;
    font-family: 'Roboto', sans-serif;
    text-align: center;
    margin-top: 0;
}

.test_result-content .ts-download {
    font-weight: 500;
    color: #fff;
    font-size: 14px;
    font-family: 'Roboto', sans-serif;
    text-align: center;
    background: #196c7e;
    border-radius: 3px;
    height: 40px;
    border: 0;
    padding: 10px 20px;
    display: inline-block;
    margin-top: 25px;
}