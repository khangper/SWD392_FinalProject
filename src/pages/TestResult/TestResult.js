import React from 'react'
import './TestResult.css'
import Header from '../../components/Header/Header'
import Footer from '../../components/Footer/Footer'
const TestResult = () => {
    return (
        <div>
            <Header />
            <div className="certification_test">
                <div className="certification_test-header">
                    <div className="certification_test-container">
                        <div className="certification_test-row">
                            <div className="certification_test-col-lg-12">
                                <div className="ct-title125">
                                    <div className="ct-titleleft">
                                        <div className="ct-ttl121">
                                            <div>
                                                <div className="ct-breadcrumb">
                                                    <p className="breadcrumb-item"><a href="index.html">Home</a></p>
                                                    <p className="breadcrumb-item"><a href="certification_center.html">Certification Center</a></p>
                                                    <p className="breadcrumb-item active" aria-current="page"><a href="certification_center.html">Result</a></p>
                                                </div>
                                            </div>
                                        </div>
                                    </div>
                                    <div className="ct-titleright">
                                        <a href="certification_center.html" className="blog_link"><i className="uil uil-angle-double-left"></i>  Back to Certification Center</a>
                                    </div>
                                </div>
                                <div className="ct-title126">
                                    <h2>Result</h2>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
                <div className="test_result-row">
                    <div className="test_result-col-md-6">
                        <div className="test_result-certi_form">
                            <div className="test_result-bg">
                                <div className="test_result-left">
                                    <div className='ts-box-icon'>
                                        <div className="result_dt">
                                            <div className='ts-tick-container'>
                                                <div className='ts-tick_icon'></div>
                                            </div>
                                            <p>Right<span>(15)</span></p>
                                        </div>
                                    </div>
                                    <div className='ts-box-icon'>
                                        <div className="result_dt">
                                            <div className='ct-cross-container'>
                                                <div className='ct-cross_icon'></div>
                                            </div>
                                            <p>Wrong<span>(5)</span></p>
                                        </div>
                                    </div>
                                    <div className='ts-box-icon'>
                                        <div className="result_dt">
                                            <div className='ts-number-container'>
                                                <h4>15</h4>
                                            </div>
                                            <p>Out of 20</p>
                                        </div>
                                    </div>

                                </div>
                                <div className="test_result-content">
                                    <h2>Congratulation! Joginder</h2>
                                    <p>You are eligible for this certificate</p>
                                    <a href="http://gambolthemes.net/html-imgs/certificate.jpg" className="ts-download" download="w3logo" target="_blank">Download Certificate</a>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>

                <Footer />

            </div>

        </div >
    )
}

export default TestResult