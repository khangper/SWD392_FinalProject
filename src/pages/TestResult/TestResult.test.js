import React from "react";
import { renderWithProviders } from "../../constant/test-util";
import TestResult from "./TestResult";
import { screen } from "@testing-library/react";

describe("TestResult", () => {
    beforeAll(() => {
        jest.spyOn(console, "error").mockImplementation(() => { });
        jest.spyOn(console, "log").mockImplementation(() => { });
    });

    afterAll(() => {
        console.error.mockRestore();
        console.log.mockRestore();
    });
    it("renders the TestResult component", async () => {
        const { container } = renderWithProviders(<TestResult />);
        expect(container).toBeDefined();
        expect(container).toMatchSnapshot();
    });
})