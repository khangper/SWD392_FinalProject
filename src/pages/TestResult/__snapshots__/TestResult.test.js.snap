// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPassword renders the ReportHistory component 1`] = `
<div>
  <div>
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="certification_test"
    >
      <div
        class="certification_test-header"
      >
        <div
          class="certification_test-container"
        >
          <div
            class="certification_test-row"
          >
            <div
              class="certification_test-col-lg-12"
            >
              <div
                class="ct-title125"
              >
                <div
                  class="ct-titleleft"
                >
                  <div
                    class="ct-ttl121"
                  >
                    <div>
                      <div
                        class="ct-breadcrumb"
                      >
                        <p
                          class="breadcrumb-item"
                        >
                          <a
                            href="index.html"
                          >
                            Home
                          </a>
                        </p>
                        <p
                          class="breadcrumb-item"
                        >
                          <a
                            href="certification_center.html"
                          >
                            Certification Center
                          </a>
                        </p>
                        <p
                          aria-current="page"
                          class="breadcrumb-item active"
                        >
                          <a
                            href="certification_center.html"
                          >
                            Result
                          </a>
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="ct-titleright"
                >
                  <a
                    class="blog_link"
                    href="certification_center.html"
                  >
                    <i
                      class="uil uil-angle-double-left"
                    />
                      Back to Certification Center
                  </a>
                </div>
              </div>
              <div
                class="ct-title126"
              >
                <h2>
                  Result
                </h2>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="test_result-row"
      >
        <div
          class="test_result-col-md-6"
        >
          <div
            class="test_result-certi_form"
          >
            <div
              class="test_result-bg"
            >
              <div
                class="test_result-left"
              >
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ts-tick-container"
                    >
                      <div
                        class="ts-tick_icon"
                      />
                    </div>
                    <p>
                      Right
                      <span>
                        (15)
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ct-cross-container"
                    >
                      <div
                        class="ct-cross_icon"
                      />
                    </div>
                    <p>
                      Wrong
                      <span>
                        (5)
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ts-number-container"
                    >
                      <h4>
                        15
                      </h4>
                    </div>
                    <p>
                      Out of 20
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="test_result-content"
              >
                <h2>
                  Congratulation! Joginder
                </h2>
                <p>
                  You are eligible for this certificate
                </p>
                <a
                  class="ts-download"
                  download="w3logo"
                  href="http://gambolthemes.net/html-imgs/certificate.jpg"
                >
                  Download Certificate
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <footer
        class="footer footer-large-container"
      >
        <div
          class="footer-links"
        >
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/about-us"
            >
              About
            </a>
            <a
              class="link"
              href="/blog-single-view/:id"
            >
              Blog
            </a>
            <a
              class="link"
              href="/career"
            >
              Careers
            </a>
            <a
              class="link"
              href="/press"
            >
              Press
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/help"
            >
              Help
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Advertise
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Developers
            </a>
            <a
              class="link"
              href="/contactus"
            >
              Contact Us
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/termsofuse"
            >
              Copyright Policy
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Terms
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Privacy Policy
            </a>
            <a
              class="link"
              href="/sitemap"
            >
              Sitemap
            </a>
          </div>
          <div
            class="footer-column"
          >
            <button
              class="teach-button"
            >
              Teach on Cursus
            </button>
            <div
              class="language-selector"
            >
              <button>
                Language 
                <span>
                  ▼
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          class="footer-bottom"
        >
          <div
            class="footer-copy"
          >
            <img
              alt=""
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
            />
            © 2024 
            <strong>
              Cursus
            </strong>
            . All Rights Reserved.
          </div>
          <div
            class="footer-social"
          >
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="facebook.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="x.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="google+.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="linkedin.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="insta.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="pinterest.png"
              />
            </a>
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;

exports[`TestResult renders the TestResult component 1`] = `
<div>
  <div>
    <header
      class="header1"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <img
        alt=""
        class="header-avatar"
        src="profile-img.jpg"
      />
    </header>
    <div
      class="certification_test"
    >
      <div
        class="certification_test-header"
      >
        <div
          class="certification_test-container"
        >
          <div
            class="certification_test-row"
          >
            <div
              class="certification_test-col-lg-12"
            >
              <div
                class="ct-title125"
              >
                <div
                  class="ct-titleleft"
                >
                  <div
                    class="ct-ttl121"
                  >
                    <div>
                      <div
                        class="ct-breadcrumb"
                      >
                        <p
                          class="breadcrumb-item"
                        >
                          <a
                            href="index.html"
                          >
                            Home
                          </a>
                        </p>
                        <p
                          class="breadcrumb-item"
                        >
                          <a
                            href="certification_center.html"
                          >
                            Certification Center
                          </a>
                        </p>
                        <p
                          aria-current="page"
                          class="breadcrumb-item active"
                        >
                          <a
                            href="certification_center.html"
                          >
                            Result
                          </a>
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="ct-titleright"
                >
                  <a
                    class="blog_link"
                    href="certification_center.html"
                  >
                    <i
                      class="uil uil-angle-double-left"
                    />
                      Back to Certification Center
                  </a>
                </div>
              </div>
              <div
                class="ct-title126"
              >
                <h2>
                  Result
                </h2>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="test_result-row"
      >
        <div
          class="test_result-col-md-6"
        >
          <div
            class="test_result-certi_form"
          >
            <div
              class="test_result-bg"
            >
              <div
                class="test_result-left"
              >
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ts-tick-container"
                    >
                      <div
                        class="ts-tick_icon"
                      />
                    </div>
                    <p>
                      Right
                      <span>
                        (15)
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ct-cross-container"
                    >
                      <div
                        class="ct-cross_icon"
                      />
                    </div>
                    <p>
                      Wrong
                      <span>
                        (5)
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="ts-box-icon"
                >
                  <div
                    class="result_dt"
                  >
                    <div
                      class="ts-number-container"
                    >
                      <h4>
                        15
                      </h4>
                    </div>
                    <p>
                      Out of 20
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="test_result-content"
              >
                <h2>
                  Congratulation! Joginder
                </h2>
                <p>
                  You are eligible for this certificate
                </p>
                <a
                  class="ts-download"
                  download="w3logo"
                  href="http://gambolthemes.net/html-imgs/certificate.jpg"
                >
                  Download Certificate
                </a>
              </div>
            </div>
          </div>
        </div>
      </div>
      <footer
        class="footer footer-large-container"
      >
        <div
          class="footer-links"
        >
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/about-us"
            >
              About
            </a>
            <a
              class="link"
              href="/blog-single-view/:id"
            >
              Blog
            </a>
            <a
              class="link"
              href="/career"
            >
              Careers
            </a>
            <a
              class="link"
              href="/press"
            >
              Press
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/help"
            >
              Help
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Advertise
            </a>
            <a
              class="link"
              href="/coming-soon"
            >
              Developers
            </a>
            <a
              class="link"
              href="/contactus"
            >
              Contact Us
            </a>
          </div>
          <div
            class="footer-column"
          >
            <a
              class="link"
              href="/termsofuse"
            >
              Copyright Policy
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Terms
            </a>
            <a
              class="link"
              href="/termsofuse"
            >
              Privacy Policy
            </a>
            <a
              class="link"
              href="/sitemap"
            >
              Sitemap
            </a>
          </div>
          <div
            class="footer-column"
          >
            <button
              class="teach-button"
            >
              Teach on Cursus
            </button>
            <div
              class="language-selector"
            >
              <button>
                Language 
                <span>
                  ▼
                </span>
              </button>
            </div>
          </div>
        </div>
        <div
          class="footer-bottom"
        >
          <div
            class="footer-copy"
          >
            <img
              alt=""
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
            />
            © 2024 
            <strong>
              Cursus
            </strong>
            . All Rights Reserved.
          </div>
          <div
            class="footer-social"
          >
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="facebook.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="x.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="google+.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="linkedin.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="insta.png"
              />
            </a>
            <a
              href="/home"
            >
              <img
                alt=""
                class="social-icon"
                src="pinterest.png"
              />
            </a>
          </div>
        </div>
      </footer>
    </div>
  </div>
</div>
`;
